src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -46,8 +46,15 @@
  */
 class AuthenticatorManager implements AuthenticatorManagerInterface, UserAuthenticatorInterface
 {
+    public const EXPOSE_NONE = 'expose_none';
+    public const EXPOSE_ACCOUNT_STATUS = 'expose_account_status';
+    public const EXPOSE_ALL = 'expose_all';
```

Comment:
@wouterj suggested using an enum in the discussion. I would also prefer that instead of string constants, as it allows more precise types for our API (yes, I know that we have lots of places in Symfony that use such string constants in places where enums would make sense, but they are generally places that were added in Symfony at a time enums were not an option for us and replacing them with an enum is not worth the BC break)